_posts/2020-07-01-#18991.md

```diff
@@ -7,16 +7,74 @@ authors: [naumenkogs]
 components: ["p2p"]
 host: naumenkogs
 status: upcoming
-commit:
+commit: 52d22a3c
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+- Today's PR is the under-review [#18991: Cache responses to GETADDR to prevent
```

Comment:
"under-review" sounds like it's under-reviewed :) 

Maybe "Today's PR is ... and is currently under review."

But you can probably just omit it since most PRs here are under review.